### PR TITLE
New gun physics and effects

### PR DESCRIPTION
**Pull request by spiral03**

### CODE REVIEW

**Comment by Amazinite** [comment]
It only triggers on reconciliation now.

File: 007_Raise Shields!/data/missions.txt [hunk 1]
```diff
@@ -72,7 +73,7 @@ event "Dragoon"
 		"Dragoon"
 			
 			
-ship "Unknown Ship" "Lionheart Prototype"
```
Comment (Amazinite) [inline 007_Raise Shields!/data/missions.txt]: Would this mess up the unknown ship in the Kestrel testing?

File: 007_Raise Shields!/data/missions.txt [hunk 2]
```diff
@@ -1,21 +1,22 @@
 mission "Dragoon Testing"
 	name "Warship Testing 2"
+	landing
 	description "Travel to the Epsilon Leonis system and meet with the Lionheart liason."
 	destination "Valhalla"
 	to offer
-		has "main plot completed" 182
```
Comment (Amazinite) [inline 007_Raise Shields!/data/missions.txt]: It only triggers on reconciliation now.

**Comment by Amazinite** [comment]
Tell me when you're done posting commits to this PR so I can merge it.

File: 007_Raise Shields!/data/missions.txt (hunk 1 above)
Comment (spiral03) [reply 007_Raise Shields!/data/missions.txt]: No, the one in the Kestrel string is called "Uknown Ship Type"

File: 007_Raise Shields!/data/missions.txt (hunk 2 above)
Comment (spiral03) [reply 007_Raise Shields!/data/missions.txt]: oops

**Comment by spiral03** [comment]
Ok, let me go over it all one more time.

**Pull request merged**